hazelcast-hibernate53/pom.xml

```diff
@@ -27,7 +27,6 @@
 
     <properties>
         <hibernate.core.version>5.3.7.Final</hibernate.core.version>
```

Comment:
Do we need this property here? should we have just one in the root pom, and up-to-date?

Reply:
removed: https://github.com/hazelcast/hazelcast-hibernate/pull/348/commits/2f5599bb41fd7540f51aa566468e42a113ea511d